src/test/unittest/utils.py

```diff
@@ -45,6 +50,33 @@ def wrapped(tc):
     return wrapped
 
 
+def require_admin(tc):
+    """
+    Enable test only if:
+    - 'enable_admin_tests' is set to 'True' in config and
+    - user has administrative privileges.
+    """
+    def util_is_admin():
+        if sys.platform == 'win32':
+            return ctypes.windll.shell32.IsUserAnAdmin() != 0
+        else:
+            return os.getuid() == 0
+
+    tc.enabled = False
+    config = configurator.Configurator().config
+    if config.enable_admin_tests:
+        if util_is_admin():
+            tc.enabled = True
+        else:
+            futils.fail('{}: FAILED: admin tests are enabled but the user'
+                        ' does not have administrative privileges'.format(tc))
+    else:
+        print('{}: SKIP: admin tests are disabled in config'
```

Comment:
`futils.skip("admin tests ...")` ?

I checked and the "SKIP" message is already generated in RUNTESTS.py:67.

Reply:
If `futils.skip()` is used, the message is not printed out ...

Reply:
Then that should be fixed instead of adding a manual print.
Are you testing with `RUNTESTS.py` or by manually running `TESTS.py`, because I can see that the message is *not* printed in the `TESTS.py` variant.